### PR TITLE
[infra] Introduce unittest_compiler_xml script

### PR DESCRIPTION
**Pull request by seanshpark**
This will introduce unittest_compiler_xml script to generate unit test xml reports

ONE-DCO-1.0-Signed-off-by: SaeHie Park <saehie.park@gmail.com>

### CODE REVIEW

**Comment by seanshpark** [comment]
For #3166, from draft #3171

**Comment by seanshpark** [comment]
`compiler_modules.sh` is introduced so that `configure_compiler_coverage.sh` can reuse this.

File: infra/scripts/unittest_compiler_xml.sh [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+NNCC_CFG_MODULES=" -DBUILD_WHITELIST=$DEBUG_BUILD_ITEMS"
+
+if [ $# -ne 0 ]; then
+	echo "Additional cmake configuration: $@"
+fi
+
+./nncc configure $NNCC_CFG_OPTION $NNCC_CFG_STRICT $NNCC_CFG_MODULES
+./nncc build -j4
+
+for TEST_BIN in `find ${ROOT_PATH}build/compiler -type f -executable -name *_test`; do
```
Comment (mhs4670go) [inline infra/scripts/unittest_compiler_xml.sh:39]:
```suggestion
for TEST_BIN in `find ${ROOT_PATH}/build/compiler -type f -executable -name *_test`; do
```

Comment (seanshpark) [reply infra/scripts/unittest_compiler_xml.sh:39]: `ROOT_PATH` has ending `/`

**Comment by hseok-oh** [comment]
Local test

```
CMake Error at /home/nfs/git/ONE/compiler/tf2tfliteV2-conversion-test/CMakeLists.txt:83 (get_target_property):
  get_target_property() called with non-existent target "tf2tfliteV2".


-- Configure TF2TFLITEV2-CONVERSION-TEST - Done
-- Configure RECORD-MINMAX-CONVERSION-TEST
-- Configure RECORD-MINMAX-CONVERSION-TEST - Done
-- Configure LUCI-VALUE-TEST
-- Configure LUCI-VALUE-TEST - Done
-- Configuring incomplete, errors occurred!
```

**Pull request merged**